src/pages/reference/python/v0/websocket/websocket-on.mdx

```diff
@@ -0,0 +1,61 @@
+export const description =
+  "Reference for Nitric's Python library - Register a handler for Websocket events."
+
+# Python - websocket.on()
```

Comment:
It's probably worth documenting in this page that all the events need to be handled by the same handler and that they're currently all _required_ to be handled.

Reply:
have made some additional notes.